v2/pgxcompat/uuid.go

```diff
@@ -0,0 +1,256 @@
+package pgxcompat
+
+import (
+	"database/sql/driver"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/jackc/pgtype"
+
+	"github.com/SKF/go-utility/v2/uuid"
+)
+
+const uuidBinaryLength = 16
+const uuidStrShortLength = 32
+const uuidStrFullLength = 36
+
+var errUndefined = errors.New("cannot encode status undefined")
+var errBadStatus = errors.New("invalid status")
+
+type UUID struct {
+	UUID   uuid.UUID
+	Status pgtype.Status
+}
+
+type Getter interface {
+	Get() interface{}
+}
+
+func (u *UUID) Set(value interface{}) error { // nolint:gocyclo
+	if value == nil {
```

Comment:
We do not need to check nil below this line so we could remove case nil as well

Reply:
Fixed